grails-app/controllers/org/pih/warehouse/inventory/InventoryItemController.groovy

```diff
@@ -320,14 +320,14 @@ class InventoryItemController {
         requisitionItems = requisitionService.getPendingRequisitionItems(location, product)
         requisitionItems.groupBy { it.requisition }.collect { k, v ->
             itemsMap.put(k, [
+                    requisitionItems: requisitionService.getListOfDetailsForPendingOutbound(v),
```

Comment:
I'm not sure if this is required, you can also access the picklist items by requsition.picklist.picklistItems.
In this case you can add here picklistItems group by lot and sum the quantity.
Also you should rename this field, those are not requisition items, it can be quantityPickedByLot

Reply:
Unfortunately, not every requisition has a picklist and in that case I'm retrieving this value from _requisitionItem.calculateQuantityPicked()_

Reply:
In case requisition has no picklist the quantity picked will be 0 and there will be no picklist items